docs/teachertool/checklists/love-meter.json

```
@@ -1,6 +1,19 @@
 {
     "name": "Love Meter",
     "criteria": [
+        {
+            "catalogCriteriaId": "499F3572-E655-4DEE-953B-5F26BF0191D7",
+            "instanceId": "b6f1ZKD6IyhuZPrrO5yhl",
+            "params": [
+                {
+                    "name": "question",
+                    "value": "Does this illustrate an understanding of how to display a random number in response to input?"
```

I think this question needs some tweaking. I know showing a number is the concrete end of the tutorial, but is it the most important that a number is displayed? What if a student decided to generate a random number and, depending on the number gotten, displayed different images? 
```suggestion
                    "value": "Does this illustrate an understanding of how to use the display in response to input?"
```

---

@kiki-lee thoughts?

---

I'd say that having something to do with the random number is most important.  It's fine as-is for now and can be tweaked if there are any changes requested by the group this afternoon.